src/index.js

```diff
@@ -218,7 +233,9 @@ class ExtractTextPlugin {
               // chunk.sortModules();
 
               async.forEach(
-                Array.from(chunk.modulesIterable),
+                Array.from(chunk.modulesIterable).sort(
+                  (a, b) => a.index - b.index
```

Comment:
This fixes the `chunk-modules-css-wrong-order` test

Reply:
Should this use `.index2` instead?

Reply:
> `index2` is the ESM evaluation order. `index` is the CJS evaluation order `@sokra` 

`css-loader@latest` returns a CJS module, so `index` should be fine for now. Please add a comment to check/switch to `index2` in case this plugin will see ESM input in the future 😛

Reply:
That's actually not up to css-loader, but up to the ETWP loader's pitching result. The pitch result is a commonjs module, so even if css-loader is converted, it won't be able to be picked by concatenated modules because it'll be required via commonjs.